app/options.html

```diff
@@ -32,7 +32,7 @@
         <input type="checkbox" id="useStartDate"><br />
         <div id="startDateShowHideWrapper" style="display:none">
             <span>Format: yyyy-MM-dd, e.g. 2012-12-31</span><br />
-            <input type="text" id="startDate"></input><br />
+            <input type="date" id="startDate"></input><br />
```

Comment:
Can you make the matching change in the _fixtures.js?
(I hate having HTML in there, btw - feel free to replace it with something better 😉)

Reply:
sure :)